lib/devise/passkeys/controllers/reauthentication_controller_concern.rb

```diff
@@ -70,7 +70,7 @@ def delete_reauthentication_challenge
         end
 
         def set_relying_party_in_request_env
-          raise "need to define relying_party for this SessionsController"
+          raise NotImplementedError, "need to define set_relying_party_in_request_env for #{self.class.name}"
```

Comment:
As annoying & confusing as it can be, there are both semantic & practical reasons to not use `NotImplentedError`, see: https://oleg0potapov.medium.com/ruby-notimplementederror-dont-use-it-dff1fd7228e5.

Some possible candidates
- Leave as-is
- `RuntimeError`
- `NoMethodError`

Reply:
Oooh, ok. Good catch. Fixed